apps/arena_load_test/lib/arena_load_test.ex

```diff
@@ -0,0 +1,5 @@
+defmodule ArenaLoadTest do
```

Comment:
We have an empty boilerplate module here

Reply:
Yeah. Other apps have it as well that's why I didn't delete it.
I can delete it if you think is better to do so.

Reply:
Yes! We can bring it back if we need it later

Reply:
Done!